packages/data-point/lib/reducer-types/reducer-list/factory.test.js

```diff
@@ -105,4 +105,12 @@ describe('factory#create', () => {
     expect(result.reducers[2].type).toBe('ReducerPath')
     expect(result.reducers[3].type).toBe('ReducerFunction')
   })
+
+  test('factory#throw error if reducer is falsy and not a string', () => {
+    expect(() => factory.create(createReducer, false)).toThrow()
+    expect(() => factory.create(createReducer, [false])).toThrow()
+    expect(() => factory.create(createReducer, [0])).toThrow()
+    expect(() => factory.create(createReducer, [undefined])).toThrow()
+    expect(() => factory.create(createReducer, ['$foo.bar', false])).toThrow()
+  })
```

Comment:
the description implies that strings are treated differently than other types, so we should have at least one test with a string as input. having a test with a falsy string `''` and another test with a string of `length > 0` is even better.

why / how are strings treated differently though?